src/olympia/blocklist/admin.py

```diff
@@ -502,9 +502,19 @@ def url_link(self, obj):
         return format_html('<a href="{}">{}</a>', obj.url, obj.url)
 
     def block_history(self, obj):
+        submission = BlockSubmission.get_submission_from_guid(obj.guid).first()
+        submission_log = (
+            format_html(
+                '<li>{date}. <a href="{url}">Changes pending</a></li>',
+                date=submission.modified.date(),
+                url=reverse(
+                    'admin:blocklist_blocksubmission_change',
+                    args=(submission.id,)))
+            if submission else '')
         return format_block_history(
             ActivityLog.objects.for_guidblock(obj.guid).filter(
-                action__in=Block.ACTIVITY_IDS).order_by('created'))
+                action__in=Block.ACTIVITY_IDS).order_by('created'),
+            additional_logs=submission_log)
```

Comment:
Should there be a test for this new code? I don't see one anywhere.

Reply:
https://github.com/mozilla/addons-server/pull/13479/files#diff-55adb0dc3ea4f62a69a70ccd80833668R1238-R1240